src/app/core/services/external-cluster.ts

```diff
@@ -86,9 +86,13 @@ export class ExternalClusterService {
     this.presetChanges.next(preset);
   }
 
+  get regionChanges() {
+    return this._regionChanges.asObservable().pipe(filter(Boolean));
```

Comment:
What if the region is cleared? I think this event will not be emitted because of `Boolean` filter. Wdyt?

Reply:
Once selected, I don't see a way to remove `regions`
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/56511988/193794784-d8be17e2-2dc2-494c-a545-9066e35284bc.png">

Reply:
The UI can change in future. I think we should remove this `filter(Boolean)` operator and handle it where we’re using `regionChanges`. That way we can also do cleanups in those places if region is cleared.